<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisFormula</class>
 <widget class="QDialog" name="AxisFormula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>234</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>445</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>123</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>222</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>415</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>X轴参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>X步长</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xStep">
              <property name="text">
               <string>3.01</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X一行数量</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="XCount">
              <property name="text">
               <string>23</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Y轴参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Y步长</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ystep">
              <property name="text">
               <string>1.25</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y一列数量</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yCount">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>自动添加点位参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>X第一行递增(0)递减(1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xorder">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X行个数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="XCount_2">
            <property name="text">
             <string>41</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>间距</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="interval">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y列个数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yCount_3">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Y递增(0)递减(1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yorder">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加一行</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>选中一行插入</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>选中行进行删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AxisFormula</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AxisFormula</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
